sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -220,7 +220,7 @@ def run(self, pipeline):
       raise ImportError(
           'Google Cloud Dataflow runner not available, '
           'please install apache_beam[gcp]')
-    self.job = apiclient.Job(pipeline.options)
+    self.job = apiclient.Job(pipeline._options)
```

Comment:
Runner.run() should be updated to take options as a separate parameter. You can add this as a TODO for now if you want.